source/chapters/appendix/mixxx_controls.rst

```diff
@@ -2506,6 +2506,19 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
 
    .. versionadded:: 2.0.0
 
+.. mixxx:control:: [ChannelN],update_replaygain_from_pregain
+                   [PreviewDeckN],update_replaygain_from_pregain
+                   [SamplerN],update_replaygain_from_pregain
+
+  Applies the deck pregain knob value to the detected ReplayGain value for the
+  current track. This is a way to update the ReplayGain value of a track if it
+  has been detected incorrectly. When this option is selected, the pregain value
+  for the deck will be adjusted so that there is no audible difference in track
+  volume, so this operation is safe to use during performance.
+
+   :range: binary
+   :feedback: ReplayGain value is updated in library, deck pregain is adjusted.
```

Comment:
```suggestion
   :feedback: ReplayGain value is updated in library, deck pregain is centered.
```

Reply:
applied manually

Reply:
ah okay, I just edited to "...deck pregain is reset to 1.0"

Reply:
done

Reply:
thx!